### PR TITLE
🚧Create hotfix templates

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] The **_head(source)_** branch for this pull request is a **_feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: feature/<issue-num>-<description>
      Example: feature/123-my-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_development_** branch with the name **_develop_**.

- [x] The **_base(target)_** branch for this pull request is a **_development_** branch with the name **_develop_**.
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_develop_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/hotfix-issue-release-template.yml [hunk 1]
```diff
@@ -0,0 +1,172 @@
+name: 🔥 HotFix
+description: Create a hotfix release issue for a project (Project maintainers only).
```
Comment (KinsonDigitalAdmin) [inline .github/ISSUE_TEMPLATE/hotfix-issue-release-template.yml:2]: Refactor to below.  Do this for all the templates

```suggestion
description: Create a hotfix release issue for the project (Project maintainers only).
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/hotfix-issue-release-template.yml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    description: DO NOT CHANGE ANYTHING BELOW!!
+    value: |
+      **_<details closed><summary>HotFix Purpose</summary>_**
+
+      The purpose of a hotfix is to release a fix to something wrong to production as soon as possible.
+      This concept should be treated very carefully.  This should only happen if the software is simply broken and
+      _**UNUSABLE**_.  If there is a simple workaround or the issue is minor, then a hotfix should not be required.
+
+      A hotfix should be a rare occasion and if it has been occurring often, then the development and release process
+      should be reviewed for improvements to try and help prevent a hotfix from occurring.
```
Comment (kselena) [inline .github/ISSUE_TEMPLATE/hotfix-issue-release-template.yml:111]: Reword  - Do you need all of this since it's pretty much described above? 
```suggestion
      If a hotfix is recurring, then the development and release process
      should be reviewed for improvements to prevent a hotfix from happening.
```

**Review by kselena — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — DISMISSED** [review]

**Review by kselena — APPROVED** [review]
